azavea/raster-foundry
# Add a --no-airflow option to ./scripts/server

## Overview

Add a `--no-airflow` option to `./scripts/server`. Allows development without starting the `airflow` service, which incurs a lot of overhead.

### Checklist


## Testing Instructions
 * run `./scripts/server --no-airflow` to start services.
 * Ensure that nginx is serving at [http://localhost:9100](http://localhost:9100).
 * In another terminal, do `docker-compose ps` and ensure that none of the `airflow-*` containers are running.



Connects #777
